src/api/app/controllers/webui/driver_update_controller.rb

```diff
@@ -116,38 +89,28 @@ def save
     file_content += "    <modules/>\n"
     file_content += "    <instsys/>\n"
     file_content += '  </packlist>'
-
-    fc.put_file file_content, opt
-
-    # find the 'generator_driver_update_disk' service
-    services = Service.find :project => @project, :package => @package
-    services = Service.new( :project => @project, :package => @package ) unless services
-
-    dud_params = []
-    dud_params << {:name => 'name', :value => @name}
-    dud_params << {:name => 'distname', :value => params[:distname]}
-    dud_params |= @architectures.map{|arch| {:name => 'arch', :value => arch}}
-    dud_params |= @repositories.map{|project| {:name => 'instrepo', :value => project}}
-
-    services.removeService( 'generator_driver_update_disk' )
-    services.addService( 'generator_driver_update_disk', -1, dud_params )
+    @package.save_file(filename: 'dud_packlist.xml', file: file_content, comment: 'Modified via driver update disk webui')
```

Comment:
TODO: Review the checking of permissions. SourceController do something different and takes care about some validations